### ISSUE TITLE
Curious

### ISSUE THREAD

**Issue by PatrickAlphaC** [0xKitsune/EVM_Mneumonic#1]
You thinking this would be more low level than huff? Just pure opcodes?

**Comment by 0xKitsune** [comment]
Yeah its basically just a framework to make it easier to write hand tuned contracts in pure opcodes. I really like Huff and this isnt meant to replace Huff at all. I wanted to build it for myself but thought others might find it useful as well so I made it public. 

Im building the project out to be very barebones, but will add some features like descriptive compilation error messages and a shell environment to play around with the opcodes [similar to the Miden repl](https://github.com/ControlCplusControlV/Scribe#miden-repl).
